### PR TITLE
Misc spelling updates

### PR DESCRIPTION
**Pull request by eerwitt**
I was reading over the code to understand how everything works and ended up making minor spelling changes. None stopped functionality for me except a change in the iPython Notebook. I hope these changes might be useful.

This is an extremely helpful utility and a cool use of DAGs.

### CODE REVIEW

File: airflow/bin/cli.py [hunk 1]
```diff
@@ -200,7 +200,7 @@ def clear(args):
     dagbag = DagBag(args.subdir)
 
     if args.dag_id not in dagbag.dags:
-        raise Exception('dag_id could not be found')
+        raise Exception('dag_id could not be fouud')
```
Comment (Mause) [inline airflow/bin/cli.py]: This creates a spelling error, rather than correcting it

**Comment by mistercrunch** [comment]
Woah. I didn't realize how much typos there was in there. I clearly need some sort of spell checker plugin in my vim. I'll merge as soon as you update the one thing @Mause pointed out. Thanks!

File: airflow/bin/cli.py (hunk 1 above)
Comment (eerwitt) [inline airflow/bin/cli.py]: :disappointed: oops sorry, good catch

**Comment by eerwitt** [comment]
None hurt my ability to use airflow, they seemed extremely minor to me. I noticed because my poor spelling caused frustration for an english as a second language colleague so I added these lines to my vimrc:

```vim
hi clear SpellBad
hi SpellBad cterm=underline
set spell spelllang=en_us
```

It works pretty well for me with PyMode and not correcting `elif`.

**Comment by mistercrunch** [comment]
Nice, I dropped those lines in my .vimrc and it seems to work well with my python config. Expect less typos from me in the future! Merging.

**Pull request merged**